### ISSUE TITLE
Fix deprecation warnings

### ISSUE THREAD

**Issue by mikegrima** [Netflix/security_monkey#976]
There are a bunch of deprecation warnings in Security Monkey:
```
/path/to/code/file/lib/python2.7/site-packages/flask_sqlalchemy/__init__.py:30: ExtDeprecationWarning: Importing flask.ext.sqlalchemy is deprecated, use flask_sqlalchemy instead.
  from flask.ext.sqlalchemy._compat import iteritems, itervalues, xrange, \
/path/to/code/file/lib/python2.7/site-packages/flask_sqlalchemy/__init__.py:30: ExtDeprecationWarning: Importing flask.ext.sqlalchemy._compat is deprecated, use flask_sqlalchemy._compat instead.
  from flask.ext.sqlalchemy._compat import iteritems, itervalues, xrange, \
/path/to/code/file/security_monkey/security_monkey/auth/modules.py:8: ExtDeprecationWarning: Importing flask.ext.login is deprecated, use flask_login instead.
  from flask.ext.login import current_user
/path/to/code/file/security_monkey/security_monkey/views/account_config.py:31: ExtDeprecationWarning: Importing flask.ext.restful is deprecated, use flask_restful instead.
  from flask.ext.restful import reqparse
/path/to/code/file/security_monkey/security_monkey/sso/views.py:15: ExtDeprecationWarning: Importing flask.ext.principal is deprecated, use flask_principal instead.
  from flask.ext.principal import Identity, identity_changed
/path/to/code/file/security_monkey/security_monkey/manage.py:18: ExtDeprecationWarning: Importing flask.ext.script is deprecated, use flask_script instead.
  from flask.ext.script import Manager, Command, Option, prompt_pass
/path/to/code/file/security_monkey/security_monkey/manage.py:27: ExtDeprecationWarning: Importing flask.ext.migrate is deprecated, use flask_migrate instead.
  from flask.ext.migrate import Migrate, MigrateCommand
```

This should be very simple to fix, and good for first time contributors.

**Comment by ArtemSokoliuk** [comment]
Mike, have you assigned it to yourself?

**Comment by mikegrima** [comment]
Feel free to work on it @ArtemSokoliuk . I'll unassign it if you want to do it.

**Comment by ArtemSokoliuk** [comment]
Rhmm, I can't assign it to myself. I'm new here, could you please help me with assigning it to me?

**Comment by mikegrima** [comment]
Hmm I can't assign to you either for some reason.  Don't worry about the assignment.

**Comment by cclauss** [comment]
https://requires.io/github/cclauss/security_monkey/requirements/?branch=develop

https://dependabot.com Auto generates pull requests to test possible package upgrades as new versions of dependencies hit PyPI.

**Comment by mikegrima** [comment]
Merged in #980
